src/main/java/cart/domain/cartitem/application/CartItemService.java

```diff
@@ -0,0 +1,92 @@
+package cart.domain.cartitem.application;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import cart.domain.cartitem.application.dto.CartItemQuantityUpdateRequest;
+import cart.domain.cartitem.application.dto.CartItemRequest;
+import cart.domain.cartitem.application.dto.CartItemResponse;
+import cart.domain.cartitem.domain.CartItem;
+import cart.domain.cartitem.persistence.CartItemDao;
+import cart.domain.member.domain.Member;
+import cart.domain.member.persistence.MemberDao;
+import cart.domain.product.persistence.ProductDao;
+import cart.global.config.AuthMember;
+import cart.global.exception.CartItemNotFoundException;
+import cart.global.exception.ProductNotFoundException;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+public class CartItemService {
+    private final ProductDao productDao;
+    private final CartItemDao cartItemDao;
+    private final MemberDao memberDao;
+
+    public CartItemService(ProductDao productDao, CartItemDao cartItemDao,
+                           MemberDao memberDao) {
+        this.productDao = productDao;
+        this.cartItemDao = cartItemDao;
+        this.memberDao = memberDao;
+    }
+
+    @Transactional(readOnly = true)
+    public List<CartItemResponse> findByMember(AuthMember authMember) {
+        Member findMember = memberDao.selectMemberByEmail(authMember.getEmail());
+        List<CartItem> cartItems = cartItemDao.findDescByMemberId(findMember.getId());
+        return cartItems.stream().map(CartItemResponse::from).collect(Collectors.toList());
```

Comment:
```suggestion
        return cartItems.stream()
        .map(CartItemResponse::from)
        .collect(Collectors.toList());
```
stream은 한줄의 하나씩 쓰는게 깔끔해 보이더라구요 :)